README.Rmd

```diff
@@ -188,8 +181,18 @@ update_reports()
 list_outputs()
 ```
 
+## Additional functionality
 
+- **`extract_string()`**: will list available outputs
```

Comment:
extract_string will or list_outputs will?

Reply:
Good spot, that was a bad copy paste job, but I have removed this code chunk anyway after having the internal/external function lightbulb moment. I have assumed that we don't necessarily want to highlight internal functions in the README as users are unlikely to use them as standalone functions. 

Happy to rework if I have missed the point and they do in fact need to be in there!

Reply:
Yep, avoid any mention of the internal functions! We don't want someone ending up with `reportfactory:::list_outputs` in some code somewhere and then getting peeved when we break it later :)